packages/next/src/build/webpack-config.ts

```diff
@@ -109,6 +109,20 @@ const mainFieldsPerCompiler: Record<CompilerNameValues, string[]> = {
   ],
 }
 
+const conditionNamesPerModuleType: Record<'edge' | 'default', string[]> = {
+  edge: [
+    'edge-light',
+    'worker',
+    'browser',
```

Comment:
You should also set them for turbopack if you are changing them.

next_edge/context.rs -> custom_conditions